lib/pr_checker.js

```diff
@@ -168,6 +170,32 @@ class PRChecker {
       }
     }
 
+    if (isFastTracked) {
+      const comment = this.comments.find((c) =>
+        FAST_TRACK_RE.test(c.bodyText));
+      if (!comment) {
+        cli.error('Unable to find the fast-track request comment.');
+        return false;
+      }
+      const [, requester] = comment.bodyText.match(FAST_TRACK_RE);
+      const collaborators = [...this.data.collaborators.values()]
+        .map((c) => c.login);
```

Comment:
nit: we can skip the intermediary array creation.
```suggestion
      const collaborators = Array.from(this.data.collaborators.values(),
                                       (c) => c.login);
```